swim/member.go

```diff
@@ -166,6 +187,23 @@ func (c Change) validateOutgoing() Change {
 	return c
 }
 
+func (c *Change) populateSubject(m *Member) {
+	if m == nil {
+		return
```

Comment:
I'd rather have a separate function that can set them back to the default values as described above. Adding it here feels a bit hacky and might be hard to understand when reading the code from the caller site.